windows/chocolatey/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,24 @@
+﻿$version = '1.7.0'
+
+$url          = "https://github.com/headsetapp/headset-electron/releases/download/v$version/HeadsetSetup.exe"
+$autoStartKey = 'HKLM:\SOFTWARE\Wow6432Node\Microsoft\Windows\CurrentVersion\Run\'
+$startKey     = 'HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\StartupApproved\Run32'
+
+$packageArgs = @{
+  packageName    = 'headset'
+  softwareName   = 'Headset'
+  fileType       = 'exe'
+  url            = $url
+  checksum       = 'NEEDS_TO_BE_REPLACED_BY_APPVEYOR!'

```

Comment:
is this a placeholder?

Reply:
Yes, this line gets changed by AppVeyor when we run `build_choco.ps1`. That script gets the `sha256sum` of the signed `exe` and puts the value there, and then builds the chocolatey packages.